spotbugs/etc/messages.xml

```diff
@@ -1675,6 +1675,20 @@ factory pattern to create these objects.</p>
             ]]>
     </Details>
   </Detector>
+  <Detector class="edu.umd.cs.findbugs.detect.FindBadEndOfStreamCheck">
+    <Details>
+      <![CDATA[
+            <p>Detector for patterns where the java.io.FileInputStream.read() or
+               java.io.FileReader.read() is converted before checking agains -1.</p>
```

Comment:
Misspelled "against". Also, the word "output" or "return value" should be added before "is converted".